addons/hr_recruitment/models/calendar.py

```diff
@@ -8,6 +8,21 @@ class CalendarEvent(models.Model):
     """ Model for Calendar Event """
     _inherit = 'calendar.event'
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        events = super(CalendarEvent, self).create(vals_list)
+        if "default_applicant_id" in self.env.context:
+            applicant_attachments = self.env['hr.applicant'].browse(self.env.context['default_applicant_id']).attachment_ids
+            for event in events:
+                self.env['ir.attachment'].sudo().create([{
+                    'name': att.name,
+                    'type': 'binary',
+                    'datas': att.datas,
+                    'res_model': event._name,
+                    'res_id': event.id
+                } for att in applicant_attachments])
```

Comment:
Aaah maybe not, there is no sudo on the `browse` so the `att.datas` would fail if the user attempt to attach an attachment he doesn't have access to.

Though given how fragile this is, I would make a unit test.

Reply:
added a read access right check on applicant, would that last version be ok ?

Reply:
why do you need the `sudo`anyway?